app/tests/util.py

```diff
@@ -0,0 +1,35 @@
+from app.models import db, User, UserSkill
+
+from flask import current_app
+from sqlalchemy.exc import IntegrityError
+
+import json
+import datetime
+
+def load_fixture(filename='sample_users.json', password='password'):
+    """
+    Populate DB from fixture data.
+    """
+
+    fixture_data = json.load(open('/noi/fixtures/%s' % filename, 'r'))
+    for i, user_data in enumerate(fixture_data):
+        skills = user_data.pop('skills')
+
+        for date_key in ['confirmed_at']:
+            user_data[date_key] = datetime.datetime.strptime(
+                user_data[date_key],
+                '%Y-%m-%d'
+            )
+
+        user = User(password=password, active=True, **user_data)
+        for name, level in skills.iteritems():
+            skill = UserSkill(name=name, level=level)
```

Comment:
I think adding a method to the User that handles the nitty gritty of creating/updating a skill is a good idea, and has the bonus of being more testable.